### DISCUSSION TITLE
Windows pairs, but soundshed does not.

### DISCUSSION THREAD

**Discussion by estes53** [soundshed/soundshed-app#34] [General]
Unable to pair in web or PC versions.
Thought I needed to be signed in, but that still did not fix it.

**Comment by webprofusion-chrisc** [comment]
It won't work for everyone but it does depend on a couple of things:
- The blue light on the back of the amp should be flashing before you start, otherwise it's paired with something else

For desktop app:
- You should pair the amp via bluetooth (Spark 40 Audio) to your computer
For web app:
- You need to pair with Spark 40 BLE, this is visible if nothing else is connected to the amp and you click the Scan button (dialog appears saying app.soundshed.com wants to pair, lists a bunch of nearby bluetooth devices). 

You mentioned web and PC versions:
 - what browser are you using (and version)
 - what OS version do you have and is it 32-bit or 64-bit
 
Note that the app just uses the available bluetooth features of your OS, it's not doing anything particularly clever outside of that. So does your bluetooth actually work and is your OS up to date.

**Comment by Brent011** [comment]
I am also having this same issue, cannot connect spark to the app or the web version. Spark amp bluetooth connection is only connected to my PC and shows Spark audio paired. This app seems to be very useful to  me and I'd like to thank everyone that has contributed to its development.

Win 10 64 bit
Chrome version 91.0.4472.114

**Reply by webprofusion-chrisc** [reply]
Thanks, when you click the 'Scan' button on the amp tab, does it show the Spark device at all? Or is it the Connect that fails?

Is your bluetooth adaptor built in or is it connected via USB? Are the bluetooth drivers up to date and what level of bluetooth is the device (e.g. 2.0, 3.0, 4.0 etc): https://www.thewindowsclub.com/how-to-check-bluetooth-version-in-windows-10

**Reply by Brent011** [reply]
Bluetooth is built in (BT 3.0) Drivers are current. Clicking scan on the app gives the never ending circle searching. The web version flashes a second with no results.

**Reply by webprofusion-chrisc** [reply]
Thanks, I believe your version of bluetooth is too old, I can't confirm the exact required level for the Spark but I think you will need at least Bluetooth version 4 (for BLE features), Bluetooth 4 started to appear on devices around 2011/2012. The web scan should find other devices even if it doesn't find the spark.

**Reply by Brent011** [reply]
Thanks for help and i agree that my bluetooth is to outdated. The newest driver that I can find for it 2010. I'll look for a diffrent solution to run the app.

**Reply by webprofusion-chrisc** [reply]
eBay has usb mini bluebooth (V5.0) adaptors for a $5 but I can't guarantee any in particular would work. If scan doesn't show devices then there's nothing that can be fixed in the app (scan features are provided by the OS etc).

**Comment by CSill2** [comment]
Amp pairs with windows but will not with the app. I have done all of the recommendations in the previous replies but no luck. BT 4.1

**Reply by webprofusion-chrisc** [reply]
What version of Windows are you using?

**Comment by CSill2** [comment]
Win 10

------ Original Message ------
From: "Christopher Cook" ***@***.***>
To: "soundshed/soundshed-app" ***@***.***>
Cc: "CSill2" ***@***.***>; "Comment" 
***@***.***>
Sent: 1/3/2022 8:51:26 PM
Subject: Re: [soundshed/soundshed-app] Windows pairs, but soundshed does 
not. (Discussion #34)

>
>What version of Windows are you using?
>
>—
>Reply to this email directly, view it on GitHub 
><https://github.com/soundshed/soundshed-app/discussions/34#discussioncomment-1903331>, 
>or unsubscribe 
><https://github.com/notifications/unsubscribe-auth/AXEDV4B2H6ZWEZS2V3PNC6TUUJHB5ANCNFSM47I5GMEQ>.
>Triage notifications on the go with GitHub Mobile for iOS 
><https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> 
>or Android 
><https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>You are receiving this because you commented.Message ID: 
***@***.***>
>

**Comment by webprofusion-chrisc** [comment]
There is a new release of the app v1.0.29 which uses a different method to connect to bluetooth and has a newer version of electron.
